app/assets/stylesheets/juntos_bootstrap/_main.scss

```diff
@@ -4244,3 +4247,37 @@ p .project-basics-icons{
 .partner-form-actions-width{
   width: $four-columns-actions-width;
 }
+
+.w-row .rewards-form-padding{
```

Comment:
since you're already using sass:

```
.rewards-form-btn-padding {
  .w-row & {
    padding: 0 0.5em;
  }
  
  &:first-child {
    padding-right: 0.5em;
  }
  
  &:last-child {
    padding-left: 0.5em;
  }
}
```